src/limits.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  Address,
+  TradingLimit,
+  TradingLimitsConfig,
+  TradingLimitsState,
+} from './types'
+
+import { Broker } from '@mento-protocol/mento-core-ts'
+import { assert } from 'console'
```

Comment:
Ah good catch, the intellisense did this automatically. 

I'll change it to `import { strict as assert } from 'assert'` which is part of Node (like in `mento.ts`)

Reply:
It's part of Node but not part of the standard browser libs so it requires SDK consumers in the browser to have that lib installed. I recommend we change it since there's no reason to add a dep just for this